config/locales/views/assignments/en.yml

```diff
@@ -28,8 +28,11 @@ en:
     hidden: "%{assignment_text} (hidden)"
     manage_course_work: Manage Course Work
     marking_scheme: 'Marking Scheme: %{identifier}'
+    maximum_file_size: 'Maximum File Size: %{file_size} MB'
     no_required_files: There are no required files for this assignment.
     none: There are currently no assignments.
+    not_only_required_files: Students can submit other files in addition to required files
+    only_required_files: Students can only submit required files below
```

Comment:
Please move this one into the same location as the other key (see above comment).

Also change text to `'You may only submit the required files listed above.'` And move this line to underneath the required files list.